techniques/general/G65.html

```diff
@@ -102,21 +98,21 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
 	
 	
 	<section id="description">
-		<h2>Description</h2>
-		<p>A breadcrumb trail (or 'breadcrumb navigation') helps the user to visualize how content has been structured and how to navigate back to previous web pages. Many even identify the current location in the series of web pages, commmonly as the last element in the trail and with a variation in its visual style. A breadcrumb trail either displays locations in the path the user took to reach the web page, or it displays the location of the current web page within the organization of the site.</p>
-		<p>Breadcrumb trails are implemented using links to the web pages that have been accessed in the process of navigating to the current web page. They are placed in the same location within each web page in the set.</p>
-		<p>It can be helpful to users to separate the items in the breadcrumb trailing with a visible separator. Examples of separators include "&gt;", "|", "/", and "→". Alternatively, one could use decorative iconography or create separators with CSS.</p>
+		<h2>解説</h2>
+		<p>パンくずリスト (又は「パンくずナビゲーション」) は、コンテンツがどのように構成されているか、前のウェブページに戻る方法を視覚化するのに役立つ。多くの場合、一連のウェブページ内の現在の場所を、パンくずリストの最後の要素として特定し、その視覚スタイルを変化させる。パンくずリストは、利用者がウェブページに到達するまでの経路上の位置を示す、又はサイトの構造内の現在のウェブページの位置を示す。</p>
```

Comment:
and を訳出しました。また、and の前後の how の訳の揃えて、読みやすくしてみました。
```suggestion
		<p>パンくずリスト (又は「パンくずナビゲーション」) は、コンテンツがどのように構成されているか、及び前のウェブページにどう戻るかを視覚化するのに役立つ。多くの場合、一連のウェブページ内の現在の場所を、パンくずリストの最後の要素として特定し、その視覚スタイルを変化させる。パンくずリストは、利用者がウェブページに到達するまでの経路上の位置を示す、又はサイトの構造内の現在のウェブページの位置を示す。</p>
```